src/FilamentMenuBuilderPlugin.php

```diff
@@ -168,6 +177,11 @@ public function getMenuPanels(): array
             ->all();
     }
 
+    public function getShowCustomLink(): bool
```

Comment:
I think named `isShowCustomLink()` is better

```suggestion
    public function isShowCustomLink(): bool
```

Reply:
Perhaps `getCustomLinkPanelShown`?

Reply:
Should keep `isShowCustomLink()`